### PR TITLE
fix(core): bypass recpatcha for functional tests

### PR DESCRIPTION
**Pull request by chanceaclark**
## What/Why?
We weren't bypassing recaptcha for functional tests when requiring it on specific pages. This was causing test failures to consistently throw.

## Testing
See the add/edit address functional test pass.

### CODE REVIEW

File: core/app/[locale]/(default)/account/(tabs)/addresses/_components/customer-edit-address.tsx [hunk 1]
```diff
@@ -4,6 +4,7 @@ import { FormFieldsFragment } from '~/client/fragments/form-fields';
 import { graphql, ResultOf } from '~/client/graphql';
 import { ExistingResultType } from '~/client/util';
 
+import { bypassReCaptcha } from '../../../../../../../lib/bypass-recaptcha';
```
Comment (deini) [inline core/app/[locale]/(default)/account/(tabs)/addresses/_components/customer-edit-address.tsx:7]: Is there a reason we don't use `~/` here?

Comment (chanceaclark) [reply core/app/[locale]/(default)/account/(tabs)/addresses/_components/customer-edit-address.tsx:7]: That was an auto import, let me update.

**Review by deini — APPROVED** [review]

**Comment by chanceaclark** [comment]
Had to update a snapshot ♻️

File: core/tests/visual-regression/components/form.spec.ts-snapshots/Form-1-tests-chromium-darwin.png
Comment (chanceaclark) [inline core/tests/visual-regression/components/form.spec.ts-snapshots/Form-1-tests-chromium-darwin.png:1]: The space for recaptcha is not being rendered anymore for tests, so had to regenerate.

**Review by deini — APPROVED** [review]

**Review by jorgemoya — APPROVED** [review]

**Pull request merged**